ergo/platforms/predictit.py

```diff
@@ -0,0 +1,242 @@
+    :ivar int id: id of the contract
+    :ivar datetime.datetime dateEnd: end-date of a market, usually None
+    :ivar str image: url of the image resource for the contract
+    :ivar str name: name of the contract
+    :ivar str shortName: shortened name of the contract
+    :ivar str status: status of the contract. Closed markets aren't included in the API, so always "Open"
+    :ivar float lastTradePrice: last price the contract was traded at
+    :ivar float bestBuyYesCost: cost to buy a single Yes share
+    :ivar float bestBuyNoCost: cost to buy a single No share
+    :ivar float bestSellYesCost: cost to sell a single Yes share
+    :ivar float bestSellNoCost: cost to sell a single No share
+    :ivar float lastClosePrice: price the contract closed at the previous day
+    :ivar int displayOrder: position of the contract in PredictIt. Defaults to 0 if sorted by lastTradePrice
+    """
+
+    def __init__(self, market: "PredictItMarket", data: Dict):
+        self.market = market
+        self._data = data
+
+    def __repr__(self):
+        return f'<PredictItQuestion title="{self.name}">'
+
+    def __getattr__(self, name: str):
+        """
+        If an attribute isn't directly on the class, check whether it's in the
+        raw contract data. If it's a time, format it appropriately.
+
+        :param name:
+        :return: attribute value
+        """
+        if name not in self._data:
+            raise AttributeError(
+                f"Attribute {name} is neither directly on this class nor in the raw question data"
+            )
+        if name != "dateEnd":
+            return self._data[name]
+        dateEnd = self._data["dateEnd"]
+        if dateEnd == "N/A":
+            return None
+        try:
+            return parse(dateEnd)
+        except ValueError:
+            print(f"The column {name} could not be converted into a datetime")
+            return dateEnd
+
+    @staticmethod
+    def to_dataframe(
+        questions: List["PredictItQuestion"], columns=None,
+    ) -> pd.DataFrame:
+        """
+        Summarize a list of questions in a dataframe
+
+        :param questions: questions to summarize
+        :param columns: list of column names as strings
+        :return: pandas dataframe summarizing the questions
+        """
+        if columns is None:
+            columns = ["id", "name", "dateEnd"]
+        data = [[question._data[key] for key in columns] for question in questions]
+
+        return pd.DataFrame(data, columns=columns)
+
+    def refresh(self):
+        """
+        Refetch the market data from PredictIt and reload the question.
+        """
+        self.market.refresh()
+        self._data = self.market.get_question(self.id)._data
+
+    def sample_community(self) -> bool:
+        """
+        Sample from the PredictIt community distribution (Bernoulli).
+
+        :return: true/false
+        """
+        return flip(self.get_community_prediction())
```

Comment:
@brachbach  I don't the best way to comment on this code generally without going to this PR, but this code just doesn't work. `self.get_community_prediction()` isn't implemented.

Reply:
I believe in a previous comment it was decided to remove get_community_prediction since it wasn't standardized across platforms. I could add it back if you thought it could be helpful, otherwise sample_community should probably be removed since it's broken currently.

Reply:
Yeah I think it's best to just remove it. @anthonyebiner do you want to make a PR for that?

Reply:
Thanks @zmpster for pointing this out!